mod.d.ts

```diff
@@ -1,96 +1,35 @@
 declare namespace BreezeRuntime {
-  export interface Env {
-    /**
-     * Retrieves the value of an environment variable.
-     * @param key The name of the environment variable.
-     * @returns The value of the environment variable, or undefined if it is not set.
-     *
-     * @example
-     * ```ts
-     * BreezeRuntime.env.get("foo");
-     * // Returns "bar" if the environment variable "foo" is set to "bar".
-     * ```
-     */
-    get(key: string): string | undefined;
-
-    /**
-     * Retrieves all environment variables.
-     * @returns A map of all environment variables.
-     *
-     * @example
-     * ```ts
-     * BreezeRuntime.env.toObject();
-     * // Returns { foo: "bar" } if the environment variable "foo" is set to "bar".
-     * ```
-     */
-    toObject(): Record<string, string | undefined>;
-
-    /**
-     * Checks whether an environment variable is set.
-     * @param key The name of the environment variable.
-     * @returns Whether the environment variable is set.
-     *
-     * @example
-     * ```ts
-     * BreezeRuntime.env.has("foo");
-     * // Returns true if the environment variable "foo" is set.
-     * ```
-     */
-    has(key: string): boolean;
+  export interface DeploymentInfo {
+    projectId: string;
+    environmentId: string;
+    functionId: string;
   }
 
   export interface Plugin {
-    /**
-     * Retrieves the plugin's endpoint.
-     *
-     * @example
-     *
-     * To retrieve the endpoint of the kv plugin:
-     * ```ts
-     * BreezeRuntime.plugins.kv.getEndpoint();
-     * // Returns new URL("breeze://kv.plugins")
-     * ```
-     *
-     * To retrieve the endpoint of the kv plugin under a subpath:
-     * ```ts
-     * BreezeRuntime.plugins.kv.getEndpoint("/put");
-     * // Returns new URL("breeze://kv.plugins/put")
-     * ```
-     */
-    getEndpoint(subpath?: string): Promise<URL>;
+    id: string;
+    endpoint: string;
   }
 
   /**
-   * The environment variables.
+   * An JWT token generated by Breeze to authorize plugin  or workflow requests.
    */
-  export const env: Env;
+  export type Token = string;
 
-  /**
-   * A map of all plugins.
-   */
-  export const plugins: Record<string, Plugin>;
+  export interface TokenOptions {
+    /**
+     * The instance name of plugin.
+     */
+    plugin: string;
```

Comment:
如果传了 token options，`plugin` 就必须填吗？
```suggestion
    plugin?: string;
```

Reply:
是的，opts 是可选的，但是如果要设置，`plugin` 是必填的。

Reply:
我想这里有没必要做成必填的呢，js 里面一般都做成 optional，这个参数和参数里面的 keys

Reply:
那需要动一下 rust 那边的代码哦，把 plugin 字段也改成 Option。